NEWS.md

```diff
@@ -3,6 +3,7 @@
 ## Next
 
 * New user option `company-tooltip-maximum-width`.
+* New frontend `company-tng'.
```

Comment:
It's more than just a frontend, right? Let's call it "new subpackage", or a "new feature" (with an explanation).

I think NEWS is one of the main sources of users finding out about new stuff in company-mode.